source/connect/connection-targets.txt

```diff
@@ -0,0 +1,116 @@
+.. _php-connection-targets:
+
+==========================
+Choose a Connection Target
+==========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, settings, client, stable api
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use a connection string and ``MongoDB\Client`` object
+to connect to different types of MongoDB deployments.
+
+.. _php-connection-atlas:
+
+Atlas
+-----
+
+To connect to a MongoDB deployment on Atlas, include the following elements
+in your connection string:
+
+- URI of your Atlas cluster
+- MongoDB username
+- MongoDB password
```

Comment:
S: just to clarify that this isn't your Atlas login; I think just saying "MongoDB" could be confusing (related to https://jira.mongodb.org/browse/DOCSP-42303)

```suggestion
- Database username
- Database user's password
```